.github/workflows/generate-frontend-build.yml

```diff
@@ -0,0 +1,73 @@
+name: Continuous Integration for Frontend
+
+on:
+  push:
```

Comment:
for this workflow, push event trigger is not required, right ?

Reply:
yup, removed it